x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -822,65 +772,18 @@ export const VisualizationWrapper = ({
         executionContext={executionContext}
         renderMode="edit"
         renderError={(errorMessage?: string | null, error?: ExpressionRenderError | null) => {
-          const errorsFromRequest = getOriginalRequestErrorMessages(error || null, core.docLinks);
+          const errorsFromRequest = getOriginalRequestErrorMessages(error || null);
           const visibleErrorMessages = errorsFromRequest.length
-            ? errorsFromRequest.map((e) => e.longMessage || e.shortMessage)
+            ? errorsFromRequest
             : errorMessage
             ? [errorMessage]
-            : [];
+            : [''];
```

Comment:
why empty string here?

Reply:
originally, WorkspaceErrors did not work with empty arrays because selecting index `0` for the active page, but this is no longer the case with `const activeError = props.errors.length ? props.errors[activePage] : '';`. Empty string removed in https://github.com/elastic/kibana/pull/172971/commits/b6ab7755be5aad86ab0215d02b4dce25491f04d1